rake/build/setup.rake

```diff
@@ -31,18 +31,17 @@ namespace :setup do
       end
     end
   end
-
-  task :install_st2_python do
-    # Only run if we use st2_python (currently only used on el6)
-    if pipeopts.st2_python == 1
-      pipeline do
-        run hostname: opts[:testnode] do |opts|
-          repo_path = '/etc/yum.repos.d/stackstorm-el6-stable.repo'
-          execute :wget, "-nv https://bintray.com/stackstorm/el6/rpm -O #{repo_path}"
-          execute :sed, "-ir 's~stackstorm/el6~stackstorm/el6/stable~' #{repo_path}"
-          execute :yum, "--nogpgcheck -y install st2python"
-        end
-      end
-    end
+ task :install_st2_python do	
```

Comment:
Looks like the indentation for this code block was just broken by this change. Please revert it.